src/pages/who-we-are/index.tsx

```diff
@@ -37,24 +39,190 @@ const EndSeparator = styled.div`
+                bg_desktop {
+                    localFile {
+                        publicURL
+                    }
+                }
+                bg_mobile {
+                    localFile {
+                        publicURL
+                    }
+                }
+            }
+            our_values {
+                header
+                values {
+                    header
+                    sub_header
+                    image {
+                        localFile {
+                            publicURL
+                        }
+                    }
+                }
+            }
+            our_principles {
+                header
+                principles {
+                    header
+                    sub_header
+                }
+                button {
+                    link_name
+                    link_url
+                }
+            }
+            our_leadership {
+                header
+                leaders {
+                    name
+                    role
+                    link_url
+                    photo {
+                        url
+                        localFile {
+                            publicURL
+                        }
+                    }
+                }
+            }
+            deriv_in_numbers {
+                header
+                sub_header
+                numbers {
+                    description
+                    number
+                }
+            }
+            our_locations {
+                header
+                numbers {
+                    description
+                    number
+                }
+                locations {
+                    country_city
+                    link_url
+                }
+                earth {
+                    localFile {
+                        publicURL
+                    }
+                }
+                earth_mobile {
+                    localFile {
+                        publicURL
+                    }
+                }
+            }
+            banner {
+                header
+                sub_header
+                link_name
+                link_url
+                bg_desktop {
+                    localFile {
+                        publicURL
+                    }
+                }
+                bg_desktop_rtl {
+                    localFile {
+                        url
+                        publicURL
+                    }
+                }
+                bg_mobile {
+                    localFile {
+                        url
+                        publicURL
+                    }
+                }
+                bg_mobile_rtl {
+                    localFile {
+                        url
+                        publicURL
+                    }
+                }
+            }
+            slider_medias {
+                slider_1 {
+                    localFile {
+                        publicURL
+                    }
```

Comment:
We can create graphql fragment for repeatable fields.

Reply:
Good point 
![Screenshot 2023-08-30 at 09 05 27](https://github.com/binary-com/deriv-com/assets/103182473/9ba2a3e5-3520-461f-a966-8b6f4580f054)

Reply:
@hasan-deriv there are too many problems with using a fragment in this case, starting with typing, continuing with Graphql validation, ending with the inability to determine and find this fragment during assembly, I tried, it works too unstable for such a minor improvement.